tests/core/engine_adapter/integration/docker/trino/catalog/datalake.properties

```diff
@@ -1,11 +1,13 @@
 connector.name=hive
+
 hive.metastore.uri=thrift://trino-datalake-hive-metastore:9083
-hive.s3.endpoint=http://minio:9000
-hive.s3.path-style-access=true
-hive.s3.aws-access-key=minio
-hive.s3.aws-secret-key=minio123
-hive.metastore-cache-ttl=0s
-hive.metastore-refresh-interval=5s
 hive.metastore.thrift.client.connect-timeout=10s
 hive.metastore.thrift.client.read-timeout=10s
 hive.storage-format=PARQUET
+
+fs.native-s3.enabled=true
```

Comment:
Trino as of [Release 458](https://trino.io/docs/current/release/release-458.html) disabled `hive.s3` ("legacy s3") and requires the use of "native s3".